includes/class-gp-notifications.php

```diff
@@ -0,0 +1,463 @@
+			// Set the emails array as empty if one admin has a comment in the thread.
+			if ( true !== empty( array_intersect( $emails, $emails_from_the_thread ) ) || ( in_array( $comment->comment_author_email, $emails ) ) ) {
+				$emails = array();
+			}
+			return $emails;
+		} catch ( Exception $e ) {
+			return array();
+		}
+	}
+
+	/**
+	 * Sends an email to all the email addresses.
+	 *
+	 * @since 0.0.2
+	 *
+	 * @param WP_Comment $comment      The comment object.
+	 * @param array      $comment_meta The meta values for the comment.
+	 * @param array      $emails       The emails that will receive the notification.
+	 *
+	 * @return bool Whether the email has been sent or not.
+	 */
+	public static function send_emails( WP_Comment $comment, array $comment_meta, array $emails ): bool {
+		/**
+		 * Filters the email addresses before sending the notifications.
+		 *
+		 * @since 0.0.2
+		 *
+		 * @param array $emails The emails to be notified.
+		 */
+		$emails = apply_filters( 'gp_notification_before_send_emails', $emails );
+		if ( ( null === $comment ) || ( null === $comment_meta ) || ( empty( $emails ) ) ) {
+			return false;
+		}
+		$emails  = self::remove_commenter_email( $comment, $emails );
+		$headers = array(
+			'Content-Type: text/html; charset=UTF-8',
+		);
+		/**
+		 * Filters the email headers.
+		 *
+		 * @since 0.0.2
+		 *
+		 * @param array $headers The email headers.
+		 */
+		$headers = apply_filters( 'gp_notification_email_headers', $headers );
+		$subject = esc_html__( 'New comment in a translation discussion' );
+		$body    = self::get_email_body( $comment, $comment_meta );
+		foreach ( $emails as $email ) {
+			$headers[] = 'Bcc: ' . $email;
+		}
+		wp_mail( '', $subject, $body, $headers );
+		return true;
+	}
+
+	/**
+	 * Creates the email body message.
+	 *
+	 * @since 0.0.2
+	 *
+	 * @param WP_Comment $comment      The comment object.
+	 * @param array      $comment_meta The meta values for the comment.
+	 *
+	 * @return string|null
+	 */
+	public static function get_email_body( WP_Comment $comment, array $comment_meta ): string {
+		$project  = self::get_project_to_translate( $comment );
+		$original = self::get_original( $comment );
+		$output   = '';
+		/**
+		 * Filters the content of the email at the beginning of the function that gets its content.
+		 *
+		 * @since 0.0.2
+		 *
+		 * @param string     $output       The content of the email.
+		 * @param WP_Comment $comment      The comment object.
+		 * @param array      $comment_meta The meta values for the comment.
+		 */
+		$output  = apply_filters( 'gp_notification_pre_email_body', $output, $comment, $comment_meta );
+		$output .= esc_html__( 'Hi:' );
+		$output .= '<br><br>';
+		$output .= esc_html__( 'There is a new comment in a discussion in the GlotPress translation system installed at ' );
+		$output .= gp_plugin_url();
+		$output .= esc_html__( ' that may be of interest to you.' );
```

Comment:
In translations, you shouldn't use assumptions of English word positions.
```suggestion
		$output .= esc_html( sprintf( __( 'There is a new comment in a GlotPress discussion at %s that may be of interest to you.', 'glotpress' ), gp_plugin_url() ) );
```